app/src/main/java/gov/hhs/cdc/trustedintermediary/external/slf4j/Slf4jLogger.java

```diff
@@ -39,31 +57,42 @@ public void logInfo(String infoMessage, Object... parameters) {
 
     @Override
     public void logWarning(String warningMessage) {
-        LOGGER.atWarn().log(() -> ANSI_YELLOW + warningMessage + ANSI_RESET);
+        getLoggingEventBuilder(Level.WARN, warningMessage).log();
     }
 
     @Override
-    public void logTrace(String traceMessage) {
-        LOGGER.atTrace().log(() -> ANSI_PURPLE + traceMessage + ANSI_RESET);
+    public void logError(String errorMessage) {
+        getLoggingEventBuilder(Level.ERROR, errorMessage).log();
     }
 
     @Override
-    public void logDebug(String debugMessage) {
-        LOGGER.atDebug().log(() -> ANSI_CYAN + debugMessage + ANSI_RESET);
+    public void logError(String errorMessage, Throwable e) {
+        getLoggingEventBuilder(Level.ERROR, errorMessage).setCause(e).log();
     }
 
     @Override
-    public void logError(String errorMessage) {
-        LOGGER.atError().log(() -> ANSI_RED + errorMessage + ANSI_RESET);
+    public void logFatal(String fatalMessage) {
+        getLoggingEventBuilder(Level.FATAL, fatalMessage).log();
     }
 
     @Override
-    public void logDebug(String debugMessage, Throwable e) {
-        LOGGER.atDebug().setMessage(() -> ANSI_CYAN + debugMessage + ANSI_RESET).setCause(e).log();
+    public void logFatal(String fatalMessage, Throwable e) {
+        getLoggingEventBuilder(Level.FATAL, fatalMessage).setCause(e).log();
     }
 
-    @Override
-    public void logError(String errorMessage, Throwable e) {
-        LOGGER.atError().setMessage(() -> ANSI_RED + errorMessage + ANSI_RESET).setCause(e).log();
+    private LoggingEventBuilder getLoggingEventBuilder(Level level, String message) {
+        return switch (level) {
+            case TRACE -> LOGGER.atTrace().setMessage(() -> ANSI_PURPLE + message + ANSI_RESET);
+            case DEBUG -> LOGGER.atDebug().setMessage(() -> ANSI_CYAN + message + ANSI_RESET);
+            case INFO -> LOGGER.atInfo().setMessage(() -> ANSI_GREEN + message + ANSI_RESET);
+            case WARN -> LOGGER.atWarn().setMessage(() -> ANSI_YELLOW + message + ANSI_RESET);
+            case ERROR -> LOGGER.atError().setMessage(() -> ANSI_RED + message + ANSI_RESET);
+            case FATAL -> {
```

Comment:
I like the enum and method. What are your thoughts on adding a default case that will let us know it didn't match any of the cases?